structure/struct.py

```diff
@@ -604,8 +604,12 @@ def get_league_matches(self):
                 match = Match(
                     match_id, radiant_team_id, dire_team_id, self.league_id, radiant_win
                 )
-                match.get_match_data()
-                self.add_match(match)
+                try:
+                    match.get_match_data()
+                    self.add_match(match)
+                except (TypeError, KeyError):
+                    # done ofr skip broken data
```

Comment:
ofr?

Reply:
need to build new version, becuse NN retraind on extended dataset. need to rebuild it after merge

Reply:
i mean on docker

Reply:
fixed